pkg/versions/1_0/txnprocessor/txnprocessor.go

```diff
@@ -50,17 +52,43 @@ func New(providers *Providers) *TxnProcessor {
 }
 
 // Process persists all of the operations for the given anchor.
-func (p *TxnProcessor) Process(sidetreeTxn txn.SidetreeTxn) error {
+func (p *TxnProcessor) Process(sidetreeTxn txn.SidetreeTxn, suffixes ...string) error {
```

Comment:
I guess this change: https://github.com/trustbloc/sidetree-core-go/pull/575/files#r600898184 was a placeholder this change.

Reply:
yes